medical-portal/src/UI/medical-portal/src/app/user-profile/manage-medical-practitioner-association-dialog/manage-medical-practitioner-association-dialog.component.ts

```diff
@@ -2,27 +2,27 @@ import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-manage-medical-practitioner-association-dialog',
-  templateUrl: './manage-medical-practitioner-association-dialog.component.html',
-  styleUrls: ['./manage-medical-practitioner-association-dialog.component.scss']
+  templateUrl:
+    './manage-medical-practitioner-association-dialog.component.html',
+  styleUrls: [
+    './manage-medical-practitioner-association-dialog.component.scss',
+  ],
 })
-export class ManageMedicalPractitionerAssociationDialogComponent{
+export class ManageMedicalPractitionerAssociationDialogComponent {
   displayedColumns: string[] = ['fullName', 'role', 'lastActive'];
   dataSource = [
-    { id: "1", fullName: "Dr. Rajan Mehra", role: "Medical Practioner", lastActive: "October 5, 2021" },
+    {
+      id: '1',
+      fullName: 'Dr. Rajan Mehra',
+      role: 'Medical Practitioner',
+      lastActive: 'October 5, 2021',
```

Comment:
Is this meant to be the current user?  If so you should be able to use the users/current service to get this.  It is probably also in the page context

Reply:
It is not the current user it is the selected Medical Practitioner Association.